input/input.c

```diff
@@ -163,14 +163,14 @@ static int openInputDevice(lua_State* L)
 
 static int closeInputDevices(lua_State* L __attribute__((unused)))
 {
-    for (size_t i = 0U; i < num_fds; i++) {
+    for (size_t i = num_fds - 1; i >= 0; i--) {
```

Comment:
Needs to be a `ssize_t` (size_t is unsigned, that makes the comparison useless, and, well, breaks it ;p).